incubator/burrow/templates/_helpers.tpl

```diff
@@ -0,0 +1,21 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "burrow.fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if ne $name .Release.Name -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s" $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "burrow.name" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" $name .Values.chain.ID | trunc 63 | trimSuffix "-" -}}
```

Comment:
This is pretty non-standard. I don't like this. Why do you include the chain id here?

Reply:
I agree this is non-standard. 

It was built in that manner because most of our users run multiple blockchains within a single k8s cluster and leverage the `chain.id` to keep them tracked properly. 

I've changed it to the standard name convention and updated the README to note usage of the nameOverride variable when running multiple chains within a cluster.